core/src/main/java/org/ironjacamar/core/connectionmanager/pool/dflt/DefaultManagedConnectionPool.java

```diff
@@ -286,6 +317,70 @@ else if (cl.getState() == DESTROY)
    }
 
    /**
+    * {@inheritDoc}
+    */
+   @Override
+   public void increaseCapacity()
+   {
+      // We have already created one connection when this method is scheduled
```

Comment:
I think we should also check for prefill credentials here.

Only the prefill credential MCP should be affected by the incrementer